lib_nbgl/include/nbgl_use_case_v2.h

```diff
@@ -0,0 +1,162 @@
+
+/*
+ * Structure for more generic navigation
+ * the app specify a list of contents which can be of mixed types for example:
+ * - a CENTERED_INFO
+ * - then a TAG_VALUE_LIST
+ * - then a INFO_LONG_PRESS
+ */
+
+typedef void (*nbgl_contentCallback_t)(uint8_t contentIndex, nbgl_content_t *content);
+
+typedef struct {
+    const nbgl_content_t      *contentsList; ///< array of nbgl_content_t (nbContents items). If NULL, callback is used instead
+    nbgl_contentCallback_t     contentGetterCallback;  ///< function to call to retrieve a given content
+    uint8_t                    nbContents;  ///< number of contents
+} nbgl_genericContents_t;
+
+
+/*
+ * Generic Nav, should be used only in non-nominal cases (for example sub-settings levels)
+ */
+
+// API to polish, but main idea is to specify an array of contents and a navigation configuration
+// like touchable, progressIndicator and all...
+// this should be used in apps:
+// - that were previously using nbgl_useCaseSettings for that
+// - that have custom content to display that doesn't fit classical cases?
+void nbgl_useCaseGenericNav(const char                   *title,
+                            const nbgl_genericContents_t *contents,
+                            uint8_t                       initPage,
+                            bool                          touchable,
```

Comment:
`touchable` is not really the right wording

Reply:
Yeap as mentioned above, the API need to be improved